### DISCUSSION TITLE
Flash funktioniert nicht

### DISCUSSION THREAD

**Discussion by Schwobi** [tbnobody/OpenDTU#1113] [General]
Ich komme leider nicht weiter. Das ESP32 Flash Download Tool (V3.9.5) startet und ich sehe auch die "Dots" im Terminal-Fenster. 
Danach drücke ich einmal den Start Button (=rechter Knopf neben dem Micro-USB-Port) und dann laufen die "Dots" einfach weiter. 
Was mache ich falsch? Habe schon alle .bin-Dateien (https://github.com/tbnobody/OpenDTU/releases) durchprobiert. 

![grafik](https://github.com/tbnobody/OpenDTU/assets/138526838/82099622-25dd-41b7-b0e1-9e6384172afb)

**Comment by tbnobody** [comment]
hast du den korrekten seriellen port ausgewählt? com1 erscheint mir etwas unrealistisch

**Reply by Schwobi** [reply]
Mir wird nichts anderes vorgeschlagen in dem Feld.

**Reply by Schwobi** [reply]
Oder benötige ich dafür noch einen Treiber?! 
![grafik](https://github.com/tbnobody/OpenDTU/assets/138526838/e73cf72b-d5cc-47a3-88e5-1527fdd266df)

**Reply by tbnobody** [reply]
Ja, der würde in der Tat helfen :)

**Reply by Schwobi** [reply]
Peinlich. Danke! ;-) 
![grafik](https://github.com/tbnobody/OpenDTU/assets/138526838/fd9c4a43-4dc5-41a2-8ee1-97303eee43f2)
